runtime/tests/checker/initialization_test.go

```diff
@@ -1,7 +1,7 @@
 /*
  * Cadence - The resource-oriented smart contract programming language
  *
- * Copyright 2019-2020 Dapper Labs, Inc.
+ * Copyright 2019-2022 Dapper Labs, Inc.
```

Comment:
Aside: is there a tool that updates copyrights or did you do this manually?

Reply:
I just saw it was outdated. I opened #1529 to make all copyright notices consistent